doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -197,8 +197,8 @@ nRF IEEE 802.15.4 radio driver
 Thread
 ------
 
-|no_changes_yet_note|
-
+* Added the new architecture option to use the OpenThread stack directly to communicate with the IEEE 802.15.4 radio driver.
+  See the :ref:`openthread_stack_architecture` user guide for more information.
```

Comment:
I would like information in the changelog (about the size differences i.e. optimization in kB) - e.g. for representative sample which could be CLI.

Reply:
I will describe the percentage difference